Update issue templates (#4671)
patternfly/patternfly: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request

> about: Suggest a new feature for Patternfly

Reviewer: ```suggestion
about: Suggest a new feature for PatternFly
```

title: "[Component] - [short description]"
labels: Feature, needs triage
assignees: ''

---

**Is this a new component or an extension of an existing one?**
What is the existing component, if any?

**Describe the feature**

> A clear and concise description of what you would like. What is the expected behavior?

Reviewer: Maybe "what you would like to see" or "the feature/enhancement"?

```suggestion
A clear and concise description of the feature/enhancement. What is the expected behavior?
```
